custom/src/main/java/com/solarwinds/opentelemetry/extensions/SolarwindsAgentListener.java

```diff
@@ -152,10 +153,16 @@ protected MetricsMonitor buildMetricsMonitor() {
                                       configs,
                                       SolarwindsInboundMetricsSpanProcessor
                                           .buildSpanMetricsCollector());
-                              metricsCollector.addCollector(
-                                  MetricsCategory.TRACING_REPORTER,
-                                  new TracingReporterMetricsCollector(
-                                      ReporterProvider.getEventReporter()));
+
+                              if (ConfigurationLoader.shouldUseOtlpForMetrics()) {
+                                metricsCollector.removeCollector(MetricsCategory.LAYER_COUNT);
+                              } else {
+                                metricsCollector.addCollector(
```

Comment:
Only add this if not `OTLP`. This is for exporting trace reporter stats metric which I'm assuming is safe to ignore.